src/components/cart/CartSidebar/cart-sidebar.scss

```diff
@@ -1,3 +1,63 @@
-.cart-sidebar [data-cart-checkout-button] {
+.cart-sidebar {
+  [data-cart-checkout-button] {
+    width: 100%;
+  }
+}
+
+.cart-sidebar__body {
+  --cart-summary-footer-height: 170px;
+
+  height: calc(100vh - var(--cart-summary-footer-height));
+  overflow-x: hidden;
+  overflow-y: auto;
+  background-color: var(--bg-neutral-light);
+
+  &.cart-sidebar__body--empty {
+    background-color: var(--bg-neutral-lightest);
+  }
+}
+
+.cart-sidebar__header {
+  position: relative;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: var(--space-5) var(--page-padding-phone) var(--space-4);
+
+  button {
+    position: absolute;
```

Comment:
Why do we need it to be `absolute`? The button seems displaced.

Reply:
because of it's visual alignment. (The outside padding is from modal body)

![image](https://user-images.githubusercontent.com/3356699/151245930-ba8aef94-beb2-4c33-8b79-11f5946c3cf6.png)